diddiparser2/parser.py

```diff
@@ -49,6 +49,15 @@ def runfile(self):
             print("=" * 60)
         success_message()
 
+    def print_command(self, cmd):
+        "By default, we use the fancy `messages.show_command`"
+        # The following line is just to fix a black <--> isort
+        # conflict:
+        from diddiparser2.messages import show_command

```

Comment:
This line exists to resolve a conflict between `black` and `isort`. `black` reformatted the imports, but `isort` wasn't satisfied, and so on.

Reply:
This went unecessary, since commits 49eb0a3 and 0e23193.